pipeline.py

```diff
@@ -28,29 +32,48 @@ class MorphologicalToken:
     Stores language params for each processed token
     """
     def __init__(self, original_word, normalized_form):
-        pass
+        self.original_word = original_word
+        self.normalized_form = normalized_form
+        self.mystem_tags = ''
+        self.pymorphy_tags = ''
 
     def __str__(self):
-        return "MorphologicalToken instance here"
+        return f"{self.normalized_form}<{self.mystem_tags}>({self.pymorphy_tags})"
+
+    def public_method(self):
```

Comment:
again and again, API cannot be changed to satisfy any style checks (!) Style check is a utility thing, while API is customer-visible. think again how to overcome the issue. maybe changing pylint settings? or making it ignore the issue? maybe google a bit for more alternatives?